packages/visx-wordcloud/src/Wordcloud.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import d3Cloud from 'd3-cloud';
+import { Group } from '@visx/group';
+import { BaseDatum, useWordcloud, WordcloudConfig } from './useWordcloud';
+
+export interface WordcloudProps<Datum extends BaseDatum> extends WordcloudConfig<Datum> {
+  children: (words: d3Cloud.Word[]) => React.ReactNode;
+}
+
+export default function Wordcloud<Datum extends BaseDatum>(props: WordcloudProps<Datum>) {
+  const { children, ...wordcloudConfig } = props;
+  const words = useWordcloud(wordcloudConfig);
+
```

Comment:
Can make it a bit more readable with.

```ts
const { width, height } = wordcloudConfig;
```

and refer to `width` and `height` afterwards.